src/applications/edu-benefits/complaint-tool/config/form.js

```diff
@@ -270,15 +269,13 @@ const formConfig = {
               'view:assistance': {
                 'ui:title': 'Which military tuition assistance benefits have you used?',
                 'ui:options': {
-                  // commenting this out will fix the title formatting but break the 'required' property
-                  // showFieldLabel: true
+                  showFieldLabel: true
```

Comment:
It looks consistent with how the other forms are set up to set showFieldLabel to true.

Reply:
@cehsu I updated the screenshot to show you what it looks like if we include it.